jerry-debugger/jerry_client_ws.py

```diff
@@ -683,13 +342,167 @@ def __init__(self, address):
+            except ValueError as val_errno:
+                result = "Error: Integer number expected, %s" % (val_errno)
+                return result
+
+            if breakpoint_index in self.active_breakpoint_list:
+                breakpoint = self.active_breakpoint_list[breakpoint_index]
+                del self.active_breakpoint_list[breakpoint_index]
+                breakpoint.active_index = -1
+                self.send_breakpoint(breakpoint)
+            elif breakpoint_index in self.pending_breakpoint_list:
+                del self.pending_breakpoint_list[breakpoint_index]
+                if not self.pending_breakpoint_list:
+                    self.send_parser_config(0)
+            else:
+                result = "Error: Breakpoint %d not found" % (breakpoint_index)
+        return result
+
+    def backtrace(self, args):
+        max_depth = 0
+
+        if args:
+            try:
+                max_depth = int(args)
+                if max_depth <= 0:
+                    self.smessage += "Error: Positive integer number expected"
+            except ValueError as val_errno:
+                self.smessage += "Error: Positive integer number expected, %s" % (val_errno)
+
+        message = struct.pack(self.byte_order + "BBIB" + self.idx_format,
+                              WEBSOCKET_BINARY_FRAME | WEBSOCKET_FIN_BIT,
+                              WEBSOCKET_FIN_BIT + 1 + 4,
+                              0,
+                              JERRY_DEBUGGER_GET_BACKTRACE,
+                              max_depth)
+        self.send_message(message)
+
+    def eval(self, code):
+        self._send_string(JERRY_DEBUGGER_EVAL_EVAL + code, JERRY_DEBUGGER_EVAL)
+
+    def throw(self, code):
+        self._send_string(JERRY_DEBUGGER_EVAL_THROW + code, JERRY_DEBUGGER_EVAL)
+
+    def abort(self, args):
+        self.delete("all")
+        self.exception("0")  # disable the exception handler
+        self._send_string(JERRY_DEBUGGER_EVAL_ABORT + args, JERRY_DEBUGGER_EVAL)
+
+    def restart(self):
+        self._send_string(JERRY_DEBUGGER_EVAL_ABORT + "\"r353t\"", JERRY_DEBUGGER_EVAL)
+
+    def exception(self, args):
+        enable = 0
+        result = ''
+        if not args:
+            result = "Error: Status expected!"
+        else:
+            try:
+                enable = int(args)
+            except (ValueError, TypeError) as val_errno:
+                return "Error: Positive integer number expected, %s" % (val_errno)
+
+        if enable == 1:
+            logging.debug("Stop at exception enabled")
+            result = "Stop at exception enabled"
+            self.send_exception_config(enable)
+        elif enable == 0:
+            logging.debug("Stop at exception disabled")
+            result = "Stop at exception disabled"
+            self.send_exception_config(enable)
+        else:
+            result = "Error: Invalid input! Usage 1: [Enable] or 0: [Disable]."
+        return result
+
```

Comment:
Sorry for the nitpicks, but maybe you could rewrite this function to increase the readability. 
```py
    def exception(self, args):
        try:
            enabled = int(args)
        except (ValueError, TypeError) as val_errno:
            return "Error: Positive integer number expected, %s" % (val_errno)

        if enabled not in [0,1]:
            return "Error: Invalid input! Usage 1: [Enable] or 0: [Disable]."

        if enabled:
            logging.debug("Stop at exception enabled")
            self.send_exception_config(enable)

            return "Stop at exception enabled"

        logging.debug("Stop at exception disabled")
        self.send_exception_config(enable)

        return "Stop at exception disabled"
```